code/modules/admin/smites/bread.dm

```diff
@@ -1,12 +1,15 @@
 #define BREADIFY_TIME (5 SECONDS)
 
-/// Turns the target into bread
+/// Turns the target into bread (Bongbread if they are Bong Bong)
 /datum/smite/bread
 	name = "Bread"
 
 /datum/smite/bread/effect(client/user, mob/living/target)
 	. = ..()
-	var/mutable_appearance/bread_appearance = mutable_appearance('icons/obj/food/burgerbread.dmi', "bread")
+    if target.name == "Bong Bong"
```

Comment:
(target.name == "Bong Bong")

Need to bracket it chief

Reply:
Fixed